CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 9003.1.25
+* Designate certain config values as mandatory
+  - acl_master_token
+  - acl_token
+  - acl_agent_token
+  - advertise_addr
+  - advertise_addr_wan
+  - bind_addr
+  - encrypt
```

Comment:
What about `license_content`? From 1.10.7-ent and later, if they do not have `license_content` the agent will fail to start.

Reply:
This is partially covered here: https://github.com/Roblox/consul-cookbook/blob/master/libraries/consul_config.rb#L304.

I can postpend something like `&& Gem::Version.new(node["consul"]["version"]) > Gem::Version.new("1.10.7-ent")` to the line, but I would like to decompose that into its own change.